A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/FMC/A32NX_FMCMainDisplay.js

```diff
@@ -301,6 +302,8 @@ class FMCMainDisplay extends BaseAirliners {
         this.updateGPSMessage();
 
         this.updateDisplayedConstraints();
+
+        this.updateProgDistance(_deltaTime);
```

Comment:
You might want to use an update throttler or call this method when opening/loading the prog page.
The prog page will be refreshed every x times a second anyways.

Reply:
> update throttler

I'll look into that.